ui/src/views/tools/ManageInstances.vue

```diff
@@ -1408,11 +1408,31 @@ export default {
         })
       }
     },
+    async fetchMoreVms (obj) {
+      var params = obj.params
+      params.token = obj.response.token
+      api('listVmwareDcVms', params).then(json => {
+        const obj = {
+          params: params,
+          response: json.listvmwaredcvmsresponse
+        }
+        this.unmanagedInstances.append(obj.response.unmanagedinstance)
+        this.checkForMoreVms(obj)
```

Comment:
If we only need existing params and token, why not just set params.token here and pass params

Reply:
becasue if token is null and we call again we start over to the beginning of the list of VMs.